samples/javascript_typescript/11.qnamaker/README.md

```diff
@@ -48,11 +48,9 @@ QnA Maker enables you to power a question and answer service from your semi-stru
 One of the basic requirements in writing your own Bot service is to seed it with questions and answers. In many cases, the questions and answers already exist in content like FAQ URLs/documents, product manuals, etc. With QnA Maker, users can query your application in a natural, conversational manner. QnA Maker uses machine learning to extract relevant question-answer pairs from your content. It also uses powerful matching and ranking algorithms to provide the best possible match between the user query and the questions.
 
 # Deploy this bot to Azure
-You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on.
+You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on. Ensure you have [Node.js](https://nodejs.org/) version 8.5 or higher.
```

Comment:
Can you adapt the Deploy section to match the one used in [netcore](https://github.com/southworkscom/BotBuilder-Samples/blob/78a021e12898d684f8c48bb72c6c2ca332e62c99/samples/csharp_dotnetcore/13.basic-bot/README.md#deploy-this-bot-to-azure)?

Reply:
Solved @VictorGrycuk